src/ray/core_worker/transport/task_receiver.cc

```diff
@@ -169,19 +169,17 @@ void TaskReceiver::HandleTask(const rpc::PushTaskRequest &request,
       }
 
       if (task_spec.IsActorCreationTask()) {
-        /// The default max concurrency for creating PoolManager should
```

Comment:
If an actor is async, `pool_manager_` will never be used.

https://github.com/ray-project/ray/blob/584d826fda984bf46fe74a69d577e68d6ddfb852/src/ray/core_worker/transport/actor_scheduling_queue.cc#L175-L194